calcpay.py

```diff
@@ -54,6 +55,34 @@ def sql_to_dataframe(db_name: str, sql_query: str):
     return df
 
 
+def convert_db_to_easier_calculations(df_record):
+    """
+    Changes the record dataframe into one that is more managable to calculate the pay.
+    To be used in the calculate_pay function.
+    Assumptions: Times are in order for each given date
+
+    :param df_record: the record dataframe from the database
+    :type df_record: pandas.Dataframe
+    :return df_record: modified record dataframe
+    :rtype df_record: pandas.Dataframe
+    """
+    # first separate dataframe into 2, 1 with timein and 1 with timeout
```

Comment:
Are the comments necessary?
Usually it is better if the comments address the "why" question rather than the "what."

Reply:
I see, they do not seem necessary.